components/import-action/tab/tab.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import classNames from "classnames";
+import styles from "./tab.module.scss";
+
+export type TabProps = {
+  isActive: Boolean;
+  title: string;
+  onClick: Function;
```

Comment:
`(target: string) => void`
consider making a type for tabs (like `type Tabs = 'registry' |  ...`)